From f8edc7ea1e33a334fc8e4d4231142221892e7ab9 Mon Sep 17 00:00:00 2001
From: Sylvain Munaut <tnt@246tNt.com>
Date: Sat, 12 May 2007 21:18:55 +0200
Subject: [PATCH 01/21] powerpc: exports rheap symbol to modules

Theses can be useful in modules too. So we export them.

Signed-off-by: Sylvain Munaut <tnt@246tNt.com>
---
 arch/powerpc/lib/rheap.c |   17 +++++++++++++++++
 1 files changed, 17 insertions(+), 0 deletions(-)

diff --git a/arch/powerpc/lib/rheap.c b/arch/powerpc/lib/rheap.c
index b2f6dcc..7c5968c 100644
--- a/arch/powerpc/lib/rheap.c
+++ b/arch/powerpc/lib/rheap.c
@@ -15,6 +15,7 @@
 #include <linux/types.h>
 #include <linux/errno.h>
 #include <linux/kernel.h>
+#include <linux/module.h>
 #include <linux/mm.h>
 #include <linux/slab.h>
 
@@ -724,3 +725,19 @@ void rh_dump_blk(rh_info_t * info, rh_block_t * blk)
 	       "blk @0x%p: 0x%lx-0x%lx (%u)\n",
 	       blk, blk->start, blk->start + blk->size, blk->size);
 }
+
+
+EXPORT_SYMBOL(rh_create);
+EXPORT_SYMBOL(rh_destroy);
+EXPORT_SYMBOL(rh_init);
+EXPORT_SYMBOL(rh_attach_region);
+EXPORT_SYMBOL(rh_detach_region);
+EXPORT_SYMBOL(rh_alloc_align);
+EXPORT_SYMBOL(rh_alloc);
+EXPORT_SYMBOL(rh_alloc_fixed);
+EXPORT_SYMBOL(rh_free);
+EXPORT_SYMBOL(rh_get_stats);
+EXPORT_SYMBOL(rh_set_owner);
+EXPORT_SYMBOL(rh_dump);
+EXPORT_SYMBOL(rh_dump_blk);
+
-- 
1.5.1.2

